### PR TITLE
[FIX] OpenAppRatingPage can’t open app page in iOS AppStore

### PR DESCRIPTION
**Pull request by EugeneButusov**
Completed. With AppID validation at settings.

### CODE REVIEW

File: Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs [hunk 1]
```diff
@@ -59,7 +59,9 @@ static ProfileSettings()
 		
 		Dictionary<string, bool?> socialIntegrationState = new Dictionary<string, bool?>();
 		Dictionary<string, Dictionary<string, string>> socialLibPaths = new Dictionary<string, Dictionary<string, string>>();
-		
+
+		GUIContent soomlaSecLabel = new GUIContent("iTunes App ID [?]:", "iOS App ID given from iTunes Connect (required ).");
```
Comment (vedi) [inline Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs]: @EugeneButusov, it's not "required" in terms of profile. Please. elaborate.

File: Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs [hunk 2]
```diff
@@ -141,6 +143,22 @@ private void WriteSocialIntegrationState()
 		}
 		
 		public void OnSoomlaGUI() {
+			if (EditorUserBuildSettings.activeBuildTarget == 
+#if UNITY_5
+			    BuildTarget.iOS
+#else
+			    BuildTarget.iPhone
+#endif
```
Comment (vedi) [inline Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs:151]: @EugeneButusov, did not this (#if) pattern @Blinkq had issues with? Will it work in Unity4.5?

File: Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs (hunk 1 above)
Comment (EugeneButusov) [inline Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs]: @vedi, I didn't finish sentence. Should be "required to use OpenAppRatingPage".

File: Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs (hunk 2 above)
Comment (EugeneButusov) [inline Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs:151]: @vedi, yes, details there: http://forum.unity3d.com/threads/request-major-version-preprocessor-define.298298/#post-1965522

Comment (vedi) [inline Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs:151]: I do not see any evidences it works in Unity4.5. Did you check?

Comment (EugeneButusov) [inline Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs:151]: @vedi, I don't have Unity v4.5, only v4.6.
But `UNITY_5` is defined only in Unity v5, so for other versions flow will goes at the `else` condition (behaviour will be the same all non-v5 Unitys).

Comment (vedi) [inline Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs:151]: @EugeneButusov, oh, you're right. Sorry.

**Pull request merged**